docs/learn-about-neptune/team-management.rst

```diff
@@ -8,100 +8,103 @@ Team, project and user management
 
-Go to **Projects** view (link at the top bar on the left side) and click on **New project** button.
+Create a project
+----------------
+Neptune lets you create a project.
+
+1. Click **Project** at the top-left of the window.
+
+2. In the pane that appears, click **New project**.
 
 .. image:: ../_static/images/how-to/team-management/create-project-1.png
    :target: ../_static/images/how-to/team-management/create-project-1.png
    :alt: Go to new project panel
 
-**Step 2**
-
-Now you can set a name, color, description and project type *Public* or *Private* (:ref:`learn more <core-concepts_project-types>`).
+3. Set a name, color, description and :ref:`project type <core-concepts_project-types>` (Public or Private).
 
 .. image:: ../_static/images/how-to/team-management/create-project-2.png
    :target: ../_static/images/how-to/team-management/create-project-2.png
    :alt: Create new project
 
-When you click "Apply" - new project is created.
+4. Click **Apply**.
+
+The new project is created.
 
 .. _how-to_team-management_invite-to-project:
 
-Add user to project
--------------------
+Add users to a project
+----------------------
 .. note::
 
-    Only project Owner can invite new users to the project. Learn more about :ref:`project roles <core-concepts_user-roles_project-owner>`.
+    Only a project Owner can invite new users to the project. Learn more about :ref:`project roles <core-concepts_user-roles_project-owner>`.
+
+1. In the Project pane, click **Settings**, then click the **People** tab. 
+
+
+- The tab displays the list of project members who already belong to the project.
+- Above the list is a field in which you can type an email or username of people you want to invite. Neptune identifies names and emails for people who already have an account. 
 
-| Go to Project **Settings**, then to **People** tab. There is a list of project members.
-| Type email or username (in case they already have an account in Neptune) of user you want to invite / add.
-| Set a role and click **Invite** / **Assign**.
+2. Specify an email or username of a user you want to add.
+
+3. Set a role and click **Invite**.
 
 .. image:: ../_static/images/how-to/team-management/add-user-1.png
    :target: ../_static/images/how-to/team-management/add-user-1.png
    :alt: Add users to project
 
-| If it is project created in team organization, user is assigned to project immediately.
-| If it is project created in the individual account, user is invited to the project via mail.
-  User needs to accept invite, before you can see such person on the list above.
+
+.. note::
+
+    - If the project is created in a team organization, the user is assigned to the project immediately.
+    - If the project is created in an individual account, the user is invited to the project by email.
+    - The invited user must accept the invitation in order to appear in the list of project members.
```

Comment:
I don't think that's true for projects in team organizations. I think it's more like an assignment.

User only need to accept invitations to team organizations.

Reply:
Discussed